include/robot_fingers/solo_eight_platform_frontend.hpp

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file
+ * @brief Frontend for the Solo8 Platform
+ * @copyright 2020, Max Planck Gesellschaft.  All rights reserved.
+ * @license BSD 3-clause
+ */
+#pragma once
+
+// TODO move to a separate package to not add unnecessary dependencies to
+// robot_fingers?
+#include <robot_interfaces/finger_types.hpp>
+#include <robot_interfaces/sensors/sensor_frontend.hpp>
+
+namespace robot_fingers
+{
+/**
+ * @brief Frontend for the Solo8 Platform
+ */
+class SoloEightPlatformFrontend
```

Comment:
Do you actually need this class?  For the TriFinger platform, we have this "platform frontend" class to provide a combined and synchronised interface to both the robot itself and the cameras.  Since you don't have cameras or other external sensors here, you could probably just use the `robot_interfaces::SimpleNJointRobotTypes<8>::Frontend` directly.

Reply:
Ah, I see. I removed the frontend